.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,53 @@
+# Adapted from https://docs.github.com/en/actions/use-cases-and-examples/publishing-packages/publishing-docker-images#publishing-images-to-github-packages
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches: ['wtsh-main']
```

Comment:
I renamed the main branch on this repository to `wtsh-main` to avoid confusion with the upstream repository.